src/components/modals/TradePreviewModalGP.vue

```diff
@@ -8,14 +8,12 @@
           >
             <span>
               {{ $t('effectivePrice') }}
-            </span>
-            <span
-              v-html="
```

Comment:
This isn't `html` anymore, so I've removed it and it also fixed the spacing issue.